pkg/storaged/vdo-details.jsx

```diff
@@ -341,6 +341,6 @@ export class VDODetails extends React.Component {
 
         const content = <Block client={client} block={block} allow_partitions={false} />;
 
-        return <StdDetailsLayout client={this.props.client} alert={alert} header={header} content={content} />;
+        return <StdDetailsLayout client={this.props.client} alerts={[alert]} header={header} content={content} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-120835-5b4a3bfc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)